### PR TITLE
Delivery components

### PR DESCRIPTION
**Pull request by nexussays**

### CODE REVIEW

File: stencil.config.ts [hunk 1]
```diff
@@ -10,30 +10,28 @@ export const config: Config = {
...
-    },
-  ],
-  buildEs5: "prod",
   plugins: [
     replace({
       exclude: "node_modules/**",
       values: {
         "process.env.STRIPE_PUBLIC_KEY": process.env.STRIPE_PUBLIC_KEY ? `"${process.env.STRIPE_PUBLIC_KEY}"` : "process.env.STRIPE_PUBLIC_KEY",
-        "process.env.PIZZA_BASE_DOMAIN": process.env.PIZZA_BASE_DOMAIN ? `"${process.env.PIZZA_BASE_DOMAIN}"` : `"https://base.polls.pizza"`,
+        "process.env.PIZZA_BASE_DOMAIN": process.env.PIZZA_BASE_DOMAIN ? `"${process.env.PIZZA_BASE_DOMAIN}"` : `"z"`,
```
Comment (mojowen) [inline stencil.config.ts:18]: want to leave this as `https://base-next.polls.pizza` so easier local dev?

**Review by mojowen — APPROVED** [review]
🍕🎉

File: stencil.config.ts (hunk 1 above)
Comment (nexussays) [reply stencil.config.ts:18]: wow, no idea what even happened there

Comment (nexussays) [reply stencil.config.ts:18]: I bet it was a typo ctrl+z -.-

**Pull request merged**